content/guides/using-the-app-catalog.mdx

```diff
@@ -113,6 +113,12 @@ oOo oOO| | | | |   (| | | (_) |  _) :_
                  Telesto 0.16.1~b1fa7c5
 ```
 
+
+<Info>
+You can use nginx version 1.25 instead of version 1.15 by appending `:1.25` to the `kraft run` command. 
```

Comment:
```suggestion
You can use Nginx version 1.25 instead of version 1.15 by appending `:1.25` to the `kraft run` command.
```

You have a trailing white space.